topsort-api-v2.yml

```diff
@@ -327,8 +249,8 @@ components:
       type: object
       required:
         - rank
```

Comment:
could you add the format (actual type) of the rank. int32 + minimum of 0.

The same for `slots`.